### DISCUSSION TITLE
How do you call a concept in python

### DISCUSSION THREAD

**Discussion by Flintols** [getgauge/gauge#1974] [General]
I am wanting to run a concept for a timed period (loop again if time is less than required time). Is there a python command to run the concept?

**Comment by zabil** [comment]
At the moment a concept cannot be invoked from a python command or run seperately.

**Reply by Flintols** [reply]
is there a way to run a step? or scenario?

**Reply by zabil** [reply]
You can run a scenario (not step) by name using the [--scenario](https://manpage.gauge.org/gauge_run.html) flag

```
gauge run --scenario "Scenario name"
```

You could run this command in a loop outside gauge.

**Reply by Flintols** [reply]
thank you

**Reply by sriv** [reply]
another alternative (if you know the line no. and file name of the scenario) - `gauge run <spec file path>:<line no>`

ex: `gauge run specs/example.spec:15`
